lib/triplex/config.ex

```diff
@@ -6,7 +6,8 @@ defmodule Triplex.Config do
   - `tenant_prefix`: a prefix for all tenants.
   - `reserved_tenants`: a list of reserved tenants, which cannot be created
   through triplex APIs. The items here can be strings or regexes.
-  - `opts`: extra options to supply for the create database query for MySQL driver
+  - `mysql`: extra options to supply for the create database query for MySQL driver.
+  The SQL standard allows a DEFAULT CHARACTER SET clause in CREATE SCHEMA than are presently accepted by PostgreSQL.
```

Comment:
Sorry, so it's possible to configure that for postgres too? 🤔 Not sure if I understand this sentence.

Reply:
It's actually quoted from compatability section here from Postgres docs https://www.postgresql.org/docs/9.3/sql-createschema.html

> The SQL standard allows a DEFAULT CHARACTER SET clause in CREATE SCHEMA, as well as more subcommand types than are presently accepted by PostgreSQL.

Per my understanding, it means that those options are not supported within the create schema statement for postgres despite being considered as SQL standard.

Reply:
Well, if I understand correctly, it is saying that it does accept, so the `charset` option could be a generic one. But hey, why introducing both these options in this PR? Why not moving this change to another one?

Reply:
Agreed. Just dropped those options to be introduced in different pr.